R/prepare_input.R

```diff
@@ -134,7 +134,7 @@ prepare_input.MultiAssayExperiment <-
       NULL
     })
     
-    if (!is.null(inl$df_)) {
+    if (length(inl$df_) > 0) {
```

Comment:
`> 0` not needed, it'll evaluate to `TRUE` for any length greater than 0

Reply:
Right